spec/features/trainees/recommending_for_qts_spec.rb

```diff
@@ -31,4 +32,9 @@ def when_i_record_the_outcome_date
   def and_i_confirm_the_outcome_details
     confirm_outcome_details_page.record_outcome.click
   end
+
+  def and_the_page_has_the_correct_links
+    expect(page).to have_link("view #{trainee_name(@trainee)}", href: trainee_path(@trainee))
```

Comment:
`expect(page)` should assert against the site prism object being used in this test for consistency

Reply:
All done, created a page object for the recommended page 👍